domain-model/src/main/java/uk/gov/hmcts/cmc/domain/constraints/ValidCountyCourtJudgmentValidator.java

```diff
@@ -20,7 +20,7 @@ public void initialize(ValidCountyCourtJudgment constraintAnnotation) {
 
     @Override
     public boolean isValid(CountyCourtJudgment ccj, ConstraintValidatorContext context) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The Cyclomatic Complexity of this method "isValid" is 11 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)